### DISCUSSION TITLE
Add safeupdate extension

### DISCUSSION THREAD

**Discussion by notramo** [geldata/gel#6274] [Ideas]
https://github.com/eradman/pg-safeupdate

**Comment by raddevon** [comment]
I think we would implement this in our compiler rather than by adding the extension. Here's an issue you can track: https://github.com/edgedb/edgedb/issues/3137

Here's a comment that would also give some context around how we're considering implementing this: https://github.com/edgedb/edgedb/pull/1773#issuecomment-722615788

In short, instead of globally requiring a filter on `insert` and `delete` which would break composability, we are considering reporting it as a warning through a forthcoming linting system.
